style/headings.md

```diff
@@ -0,0 +1,30 @@
+# Headings in CoreOS Documentation
+`# Headings in CoreOS Documentation`
```

Comment:
I wasn't sure what to make of this at first. Maybe we have an example that uses the headings in the doc as an example and it looks something like:

For example, this document's headings are structured in code as:

```
# Headings in CoreOS Documentation

## Documentation Format

### Subheadings

#### Subheadings and the Sidebar Outline

### Example: *The "Average" Document*
```